### ISSUE TITLE
vmagent statefulset support

### ISSUE THREAD

**Issue by f41gh7** [VictoriaMetrics/operator#219]
Its nice to have ability to use vmagent with statefulset, it allows to add `persistentVolumeClaimTemplate` and use pvc for metrics cache.

**Comment by acukan** [comment]
Hi, instead of a hacky solution are there any plans for implementing it in operator?

**Comment by f41gh7** [comment]
Yes, it'll be added in the near future. Cannot say exactly when, next release probably.

**Comment by aberenshtein** [comment]
any progress here?

**Comment by f41gh7** [comment]
Running out of time for now(

**Comment by f41gh7** [comment]
changes was included into [v0.25.0 release](https://github.com/VictoriaMetrics/operator/releases/tag/v0.25.0)

**Comment by aberenshtein** [comment]
@f41gh7 is it updated in the docs? 
https://docs.victoriametrics.com/operator/api.html#vmagentspec

**Comment by f41gh7** [comment]
Will be updated today

**Comment by omegarus** [comment]
Helm chart needs update as well if I understand correctly ?

**Comment by f41gh7** [comment]
Chart was updated, it's still manual process (
